product_docs/docs/pgd/5/rel_notes/pgd_5.1.0_rel_notes.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Release notes for EDB Postgres Distributed version 5.1.0"
+navTitle: "Version 5.1.0"
+---
+
+EDB Postgres Distributed version 5.1.0 is a patch version of EDB Postgres Distributed.
+This version addresses security vulnerabilities in dependencies for PGD-Proxy and PGD-CLI.
```

Comment:
why are we calling it a Patch version? I thought based on semver, a patch version is when the last digit is incremented. But here we are releasing 5.1.0 (or 5.1 IOW).

Reply:
Yes, this is a minor version, not patch.

Reply:
Ok, thanks for confirming.